src/wp-admin/includes/file.php

```diff
@@ -786,6 +786,9 @@ function validate_file_to_edit( $file, $allowed_files = array() ) {
  *     @type bool     $test_size                Whether to test that the file size is greater than zero bytes.
  *     @type bool     $test_type                Whether to test that the mime type of the file is as expected.
  *     @type string[] $mimes                    Array of allowed mime types keyed by their file extension regex.
+ *     @type string[] $upload_dir               Array of the uploads directory data as returned by
+ *                                              {@see wp_upload_dir()} or {@see wp_font_dir()}. If not set the function
+ *                                              will use {@see wp_upload_dir()}.
```

Comment:
```suggestion
 *     @type string[] $upload_dir               Array of the uploads directory data as returned by
 *                                              {@see wp_upload_dir()} or {@see wp_font_dir()}. If not set the function
 *                                              will use {@see wp_upload_dir()}.
```
```suggestion
 *     @type string[] $upload_dir               Array of the uploads directory data as returned by
 *                                              wp_upload_dir() or wp_font_dir(). Defaults to wp_upload_dir().
```

The docs parser auto links functions without see annotations, it's only needed for hooks per [doc standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/) -- see section headed Other Descriptions